packages/glibc.rb

```diff
@@ -3,29 +3,31 @@
 class Glibc < Package
   description 'The GNU C Library project provides the core libraries for GNU/Linux systems.'
   homepage 'https://www.gnu.org/software/libc/'
-  version '1.1'
+  version '1.2'
 
   is_fake
 
-  if (File.exist? "/lib/libc-2.23.so") || (File.exist? "/lib64/libc-2.23.so")
-    depends_on 'glibc223'
-    # Check previous version
-    if File.exist? CREW_CONFIG_PATH + "meta/glibc219.filelist"
-      conflict_solve = "`crew remove glibc219`"
-    end
-  else
-    depends_on 'glibc219'
-  end
-
   # Check old glibc
   if File.exist? CREW_CONFIG_PATH + "meta/glibc.filelist"
-    if File.exist? CREW_CONFIG_PATH + "meta/glibc223.filelist"
-      # Already installed old glibc and glibc223, so need to remove both once
-      conflict_solve = "`crew remove glibc223 glibc`"
+    if File.exist? CREW_CONFIG_PATH + "meta/glibc219.filelist"
+      # Already installed old glibc and glibc219, so need to remove both once
+      conflict_solve = "`crew remove glibc219 glibc`"
     else
       # Already installed old glibc, so need it once
       conflict_solve = "`crew remove glibc`"
     end
+  else
+    case
+    when Integer(CHROMEOS_RELEASE) <= 66
+      depends_on 'glibc223'
+    else
+      depends_on 'glibc227'
+    end
+  end
+
+  if File.exist? CREW_CONFIG_PATH + "meta/glibc223.filelist"
+    # Already installed old glibc version 2.23, so need to remove
+    conflict_solve = "`crew remove glibc223`"
```

Comment:
I think you will want to do this only when `CHROMEOS_RELEASE` > 66, correct?

Reply:
Nope. Then it would install glibc223 if the chrome os release was greater than 66. That would break Chromebrew for everybody.

Reply:
Think about this a bit.  If you are on 66 or less and you remove glibc223, doesn't that seem wrong?  There is no conditional for CHROMEOS_RELEASE on this logic.

Reply:
Chromebrew doesn't do that much dependency checking. If you install foo, and foo depends on bar, when you uninstall foo, bar isn't uninstalled.

Reply:
It's like trying to say 6<10 and 10>6 aren't the same.

Reply:
Chromebrew doesn't uninstall dependencies that aren't used for you but neither does Debian.  There is an [issue](#858) that presents a problem if you uninstall a dependency that is required by other package.  Chromebrew currently doesn't prevent you from doing this which is something we should definitely work on fixing at some point.  Maybe you can work on that. :)